XRTK.Lumin/Packages/com.xrtk.lumin/package.json

```diff
@@ -13,13 +13,12 @@
     "reality",
     "lumin"
   ],
-  "version": "0.1.11",
+  "version": "0.2.0",
   "unity": "2019.1",
   "license": "MIT",
   "src": "Assets/XRTK.Lumin",
   "author": "XRTK Team (https://github.com/XRTK)",
   "dependencies": {
-    "com.unity.xr.magicleap": "2.0.0-preview.19",
```

Comment:
We might need this dependency, no?

Reply:
Well yeah, as long as we are on 2019.1 yes. That change originates back to when I was working on hands using 2019.2 and that dependencies kept causing me issues. It's on my "TODO before finishing the PR" list to readd that. But you can go ahead and bring it back since I am now on 2019.1 solely again.